requirements.txt

```diff
@@ -28,10 +28,11 @@ turtlethread>=0.0.6
 pygame==2.1.2
 pre-commit==2.20.0
 babel==2.14.0
-jinja-partials==0.1.1
+jinja-partials==0.1.1clea
```

Comment:
I suspect this is a typo @TiBiBa? Did you mean just to leave it at 0.1.1 or is there another version needed?

Reply:
I think this is a typo, will look into this. We should also decide on a nice test case, but that might be easy to fix on Alpha.